viite-UI/src/model/RoadCollection.js

```diff
@@ -171,7 +171,7 @@
           });
 
         var nonUnderConstructionRoadLinkGroups = _.reject(roadLinkGroups, function(group) {
```

Comment:
Should this variable name be changed since this is not "nonUnderConstructionRoadLinkGroups" any more?

Reply:
true

Reply:
Also good to make sure that on line 177 we don't concatenate same links multiple times.

Reply:
yes we dont get same links since they are being filtered beforehand